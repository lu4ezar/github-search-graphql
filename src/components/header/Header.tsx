import React from "react";
import HeaderDiv from "./styled";

const Header = () => <HeaderDiv>GITHUB SEARCH</HeaderDiv>;

export default Header;
