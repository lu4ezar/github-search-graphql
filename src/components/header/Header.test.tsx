import React from "react";
import { render } from "@testing-library/react";
import Header from "./Header";

it("matches the snapshot", () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});

it("renders app title", () => {
  const { getByText } = render(<Header />);
  const textLink = getByText("GITHUB SEARCH");
  expect(textLink).toBeInTheDocument();
});
