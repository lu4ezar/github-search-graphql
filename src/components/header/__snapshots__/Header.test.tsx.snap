// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div>
  <div
    class="sc-bdVaJa dEOUZa"
  >
    GITHUB SEARCH
  </div>
</div>
`;
